### PR TITLE
Migrate to GitHub Actions.

### PR DESCRIPTION
**Pull request by jezdez**
Travis CI has a new pricing model which places limits on open source.

* https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing
* https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works

Many projects are moving to GitHub Actions instead, including Jazzband projects:

* https://github.com/orgs/jazzband/projects/1
* https://github.com/jazzband-roadies/help/issues/206

This is based on https://github.com/jazzband/contextlib2/pull/26.

TODO:

* [x] @jezdez to add `JAZZBAND_RELEASE_KEY` to the repo secrets.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  build:
+    if: github.repository == 'jazzband/django-downloadview'
```
Comment (Natim) [inline .github/workflows/release.yml:10]: Is this really necessary? Can you explain the rationale?

File: demo/setup.py [hunk 2]
```diff
@@ -1,46 +1,27 @@
-"""Python packaging."""
+# -*- coding: utf-8 -*-
```
Comment (Natim) [inline demo/setup.py:1]: This can be removed since we now support Python 3.6+

**Review by Natim — APPROVED** [review]

**Comment by Natim** [comment]
Thank you for taking the time :pray:

File: demo/setup.py (hunk 2 above)
Comment (jezdez) [inline demo/setup.py:1]:
```suggestion
```

Comment (jezdez) [inline demo/setup.py]:
```suggestion
```

File: setup.py [hunk 3]
```diff
@@ -1,98 +1,63 @@
-#!/usr/bin/env python
-"""Python packaging."""
+# -*- coding: utf-8 -*-
```
Comment (jezdez) [inline setup.py:1]:
```suggestion
```

**Pull request merged**